packages/web-worker/src/prevent-discard.js

```diff
@@ -0,0 +1,14 @@
+const extensionRegex = /^(chrome|moz|safari|safari-web)-extension:/
+
+module.exports = {
+  name: 'preventDiscard',
+  load: client => {
+    client.addOnError(event => {
```

Comment:
As discussed earlier, this isn't appropriate for non-browser extensions — we'll need to figure out how to handle that